docs/static/openapi.yml

```diff
@@ -759,7 +759,7 @@ paths:
             type: boolean
       tags:
         - Query
-  /quasarlabs/quasarnode/qvesting/accounts:
+  /quasarlabs/quasarnode/qvesting/vesting_accounts:
```

Comment:
Did you generated the openapi.yml with  scripts/generate-docs.sh script.

Reply:
No, the script is currently not working. I can try fix it but at the moment I generated everything manually.